invenio_github/models.py

```diff
@@ -220,29 +226,32 @@ def disable(cls, user_id, github_id, name):
         :param repo_id: GitHub id of the repository.
         :param name: Fully qualified name of the repository.
         """
-        repo = cls.get(user_id, github_id=github_id, name=name)
-        if repo:
-            repo.enabled = False
-            repo.user_id = None
+        try:
+            repo = cls.get(user_id, github_id=github_id, name=name)
             repo.hook = None
-            db.session.add(repo)
+            repo.user_id = None
+        except NoResultFound:
```

Comment:
That's true but if you catch it already in model then we can not even warn user that lets say: "The repo was already disabled."